core/common/src/main/java/org/phenotips/remote/common/internal/api/DefaultPatientToJSONConverter.java

```diff
@@ -86,17 +86,15 @@ public JSONObject convert(Patient patient, boolean removePrivateData, int includ
         } catch (Exception ex) {
             // Do nothing. These are optional.
         }
+        } catch (Exception ex) {
+            // Do nothing. These are optional.
+        }
```

Comment:
I don't understand these lines. Does this even compile?

Reply:
oops, a glitch, fixed